src/main/java/net/masterthought/cucumber/generators/FailuresOverviewPage.java

```diff
@@ -16,6 +21,19 @@ public String getWebPage() {
 
     @Override
     public void prepareReport() {
-        context.put("all_features", report.getAllFeatures());
+        List<Element> failures = new ArrayList<>();
+        for (Feature feature : report.getAllFeatures()) {
+            if (feature.getStatus().isPassed()) continue;
+
+            for (Element element : feature.getElements()) {
+                if (!element.getBeforeStatus().isPassed()
+                    || !element.getStepsStatus().isPassed()
+                    || !element.getAfterStatus().isPassed()) {
```

Comment:
In that case there should be a configuration flag to mark scenarios as failed if their hooks fail. But that's a separate issue so I've created it: #497